### ISSUE TITLE
UX - zoom controls can be improved.

### ISSUE THREAD

**Issue by dlzi** [rncbc/qtractor#311]
Hi Rui,

I think the zoom controls could be improved in their appearance. It's hard to distinguish them, specially when they are grouped so near to each other. 

![Screenshot_20210811_144112](https://user-images.githubusercontent.com/1160818/129031619-95feacea-7985-4590-b7b9-a5728ee0de5b.png)

I noticed that you are using .png files for icons. With QT 6, I think you could easily use SVG, which would give you much more responsiveness and better looking icons.

**Comment by rncbc** [comment]
how could a svg make it any better when rendered to a 16x16 pixmap?
mind you, the original icon is indeed a svg alright, just that is distributed in source tree as a png.
cheers

**Comment by dlzi** [comment]
Maybe play with colours or the icons should have less stroke, so you have more room to see the "-" and "+" signs? Anyway, just some ideas here.

**Comment by oldcastlehq** [comment]
Yeah..on my laptop they are quite hard to see too. Any updates on this issue?

**Comment by rncbc** [comment]
> Yeah..on my laptop they are quite hard to see too. Any updates on this issue?

if you can hardly see them, use ctrl+mouse-wheel instead ;)

**Comment by rncbc** [comment]
not a remedy whatsoever, there are (always been) some other means to zoom in and out--take the menus (View > Zoom) for instance...

**Comment by oldcastlehq** [comment]
Sure. I understand, and I appreciate that there are more than one way of doing this. I just agreed with OP that the icons could look better. Anyway, thanks for making Qtractor.
